### PR TITLE
Allow many to many relationship (via secondary)

### LINKED ISSUE #16: Many to Many relationship
relationships declared with a `secondary` argument are not loaded correctly.
http://docs.sqlalchemy.org/en/latest/orm/basic_relationships.html#many-to-many

### PR DESCRIPTION
**Pull request by timc13**
fixes #16

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,8 @@
+0.9.0 (unreleased)
+======================
+
+- fix loading many-to-many relationships (via secondary)
```
Comment (schettino72) [inline CHANGES]: "fix" implies it was broken. But it was just not supported... this is a new feature, right?

File: tests/test_sqla_yaml_fixtures.py [hunk 2]
```diff
@@ -349,7 +375,8 @@ def test_2many(session):
 
 - Group:
   - name: Ramones
-    members: [joey.profile]
```
Comment (schettino72) [inline tests/test_sqla_yaml_fixtures.py]: Is this change necessary? If not, why change it?

File: tests/test_sqla_yaml_fixtures.py [hunk 3]
```diff
@@ -499,3 +526,29 @@ def test_custom_loader(session):
...
+  - __key__: johnny
+    username: johnny
+    email: johnny@example.com
+    friends:
+      - joey
+      - bobby
+"""
+    sqla_yaml_fixtures.load(BaseModel, session, fixture)
+    users = session.query(User).all()
+    for u in users:
```
Comment (schettino72) [inline tests/test_sqla_yaml_fixtures.py]: Please keep the tests "stupid" simple. You should not use FOR-loops or IF conditions on tests.

**Review by schettino72 — CHANGES_REQUESTED** [review]
Great thanks

**Comment by timc13** [comment]
addressed comments

**Review by schettino72 — APPROVED** [review]
Great. thanks :)

Feel free to merge and make a release.

File: tests/test_sqla_yaml_fixtures.py [hunk 4]
```diff
@@ -499,3 +526,27 @@ def test_custom_loader(session):
...
+    email: johnny@example.com
+    friends:
+      - joey
+      - bobby
+"""
+    sqla_yaml_fixtures.load(BaseModel, session, fixture)
+    users = session.query(User).order_by(User.username).all()
+    assert users[0].friends == []
+    assert users[1].friends == []
+    assert users[2].friends[0].id == users[0].id
```
Comment (schettino72) [inline tests/test_sqla_yaml_fixtures.py]: just a nitpick. I read this and I need to think which friend will be the last one.
One way to improve it would be just add one assert on the `users[2].username`, just to make it clearer.

Comment (timc13) [reply tests/test_sqla_yaml_fixtures.py]: updated commit

File: tests/test_sqla_yaml_fixtures.py [hunk 5]
```diff
@@ -499,3 +526,29 @@ def test_custom_loader(session):
...
+
+
+def test_many_2_many(session):
+    fixture = """
+- User:
+  - __key__: joey
+    username: joey
+    email: joey@example.com
+  - __key__: bobby
+    username: bobby
```
Comment (schettino72) [inline tests/test_sqla_yaml_fixtures.py:538]: oh. and there is no "Bobby" Ramone :grin: https://en.wikipedia.org/wiki/Ramones#Band_members

Comment (timc13) [reply tests/test_sqla_yaml_fixtures.py:538]: haha - I just realized the ramones reference
